internal/informer/bbp/informer.go

```diff
@@ -117,7 +118,7 @@ func (i *Informer) updateHandler(oldObj, newObj interface{}) {
 		return
 	}
 
-	if previous.Annotations[AnnotationKey] != AnnotationValue || current.Annotations[AnnotationKey] != AnnotationValue {
+	if previous.Labels[LabelKey] != LabelValue || current.Labels[LabelKey] != LabelValue {
```

Comment:
It is possibly not related to the scope of the current PR.

@yitsushi should I change this line in the current PR after the create/delete objects logic was added?

Reply:
I don't think we have to check the previous object. I don't even know if we have to check **anything** on the previous object, or we need it at all. We care about the generated `ConfigMap` with the plan linked to the `current` object, and doesn't really matter what was the previous state, we don't use values from there.

I added that check there mostly as a filler to prevent go from crying about "unused variable" and I didn't want to write `_ = previous` while tried to capture the fact we have that available.

Summary: I think we can even delete everything about the `previous` object as I don't see any reasons to keep it. Something like this:
```diff
-func (i *Informer) updateHandler(oldObj, newObj interface{}) {
+func (i *Informer) updateHandler(_, newObj interface{}) {
// ...
-	previous, ok := oldObj.(*tfv1alpha2.Terraform)
-	if !ok {
-		i.log.Info("previous object is not a Terraform object", "object", oldObj)
-
-		return
-	}
// ...
-	if previous.Annotations[AnnotationKey] != AnnotationValue || current.Annotations[AnnotationKey] != AnnotationValue {
+	if current.Annotations[AnnotationKey] != AnnotationValue {

```